### ISSUE TITLE
Podman Desktop connection issue with WSL machine

### ISSUE THREAD

**Issue by techAdamWilliams** [containers/podman-desktop#3216]
### Bug description

Upgraded from 1.1.0 to 1.2.0 on Podman Desktop, running versoin 4.5.1 of podman in a linux vm that was created under Podman 1.1.0. After upgrading podman desktop does not properly connect with the linux vm. I have to force a refresh of the containers and images however trying to run commands does not function properly.

### Operating system

Windows 10 Pro 19045.3208

### Installation Method

Installer from website/GitHub releases

### Version

1.2.0

### Relevant log output

```shell
log : main ↪️ PluginSystem: received dom-ready event from the UI
log : main ↪️ Error while fetching API groups: HttpError: HTTP request failed
log : main ↪️ Error while fetching API groups: HttpError: HTTP request failed
log : main ↪️ Delayed startup, flushing 9 events
log : main ↪️ System ready. Loading extensions...
log : main ↪️ Activating extension (podman-desktop.compose)
log : main ↪️ Activating extension (podman-desktop.docker)
log : main ↪️ Activating extension (podman-desktop.kind)
log : main ↪️ Activating extension (podman-desktop.kube-context)
log : main ↪️ [kube-context] starting extension kube-context
log : main ↪️ Activating extension (podman-desktop.lima)
error : main ↪️ [lima] Could not find podman socket at C:\Users\JFTG84\.lima\podman\sock\podman.sock
log : main ↪️ Activating extension (podman-desktop.podman)
log : main ↪️ Activating extension (podman-desktop.registries)
log : main ↪️ Activating extension (podman-desktop.compose) ended
log : main ↪️ Activating extension (podman-desktop.docker) ended
log : main ↪️ Activating extension (podman-desktop.lima) ended
log : main ↪️ Activating extension (podman-desktop.kube-context) ended
log : main ↪️ Activating extension (podman-desktop.registries) ended
debug : main ↪️ [compose] Error while pinging docker Error: connect ENOENT //./pipe/docker_engine
log : main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 283.64669999992475ms
log : main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 174.90090000000782ms
debug : main ↪️ [podman] Error while pinging docker as podman Error: connect ENOENT //./pipe/docker_engine
error : main ↪️ (node:24224) UnhandledPromiseRejectionWarning: Error: EINVAL: invalid argument, stat '//./pipe/'
(Use `Podman Desktop --trace-warnings ...` to show where the warning was created)
error : main ↪️ (node:24224) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
error : main ↪️ [kind] Error: '"kind-windows-amd64"' is not recognized as an internal or external command,
operable program or batch file.

log : main ↪️ Activating extension (podman-desktop.kind) ended
debug : main ↪️ [podman] Error while pinging docker as podman Error: connect ENOENT //./pipe/docker_engine
error : main ↪️ [podman] Error parsing auth file SyntaxError: Unexpected end of JSON input
log : main ↪️ Activating extension (podman-desktop.podman) ended
log : main ↪️ PluginSystem: initialization done.
log : main ↪️ Autostarting container engine
log : main ↪️ [podman] Podman extension: Autostarting machine podman-machine-default
log : main ↪️ Fetched https://registry.podman-desktop.io/api/extensions.json in 137.05529999989085ms
debug : main ↪️ [podman] Error while pinging docker as podman Error: connect ENOENT //./pipe/docker_engine
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
log : main ↪️ event is [object Object]
```

**Comment by benoitf** [comment]
Hello @techAdamWilliams could you check your podman auth.json file. It looks like it has a parsing issue from the log you reported.

**Comment by benoitf** [comment]
there should be a bulb icon 💡 in the status bar

Could you try to check the stores and see what events are reported

**Comment by benoitf** [comment]
I think this one is solved with 1.2.1 release
